recipes/xilinx-runtime/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/env/bash
+set -e
+
+mkdir -p $SRC_DIR/build-release
+
```

Comment:
It would be really nice if upstream could be patched to correctly include the generated version artifacts in the release tarballs so that the version information is correct when building from tarball.

Reply:
> It would be really nice if upstream could be patched to correctly include the generated version artifacts in the release tarballs so that the version information is correct when building from tarball.

I opened this issue on your behalf: https://github.com/Xilinx/XRT/issues/6794